omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -45,8 +45,30 @@
       alias /opt/opscode/version-manifest.txt;
     }
 
-    <% if node['private_chef']['data_collector']['proxy'] -%>
+    <% if node['private_chef']['data_collector']['root_url'] && node['private_chef']['data_collector']['token']-%>
```

Comment:
The old configuration value needs to work still or this will break people using the proxying feature.